README.md

```diff
@@ -3,16 +3,104 @@ Sunrise Java CMS :sunrise: :pencil2:
 
 [![Build Status](https://travis-ci.org/commercetools/commercetools-sunrise-java-cms.png?branch=master)](https://travis-ci.org/commercetools/commercetools-sunrise-java-cms) [![Stories in Ready](https://badge.waffle.io/commercetools/commercetools-sunrise-java-cms.png?label=ready&title=Ready)](https://waffle.io/commercetools/commercetools-sunrise-java-cms)
 
-Module for [Sunrise Java](https://github.com/sphereio/commercetools-sunrise-java) with different supported services to use CMS. 
+Module for [Sunrise Java](https://github.com/sphereio/commercetools-sunrise-java)
+defining unified API to fetch content from different CMS.
 
 * [Javadoc](https://commercetools.github.io/commercetools-sunrise-java-cms/javadoc/index.html)
 
 ##Installation
-Choose the CMS Service you want to use and follow its installation details:
+Choose the CMS adapter and follow its installation instructions:
 
 - [Contentful Sunrise Adapter](/cms-contentful)
 
-Note that if you are not using a Sunrise-based project, you'll additionally need to use as dependency:
```

Comment:
I think this is useful information and would suggest to keep it together with the removed line below (originanly at line #17).